src/cart.html

```diff
@@ -0,0 +1,120 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>E Tutor - Cart Page</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <!-- navbar -->
+    <nav class="navbar flex-center-y flex-space-between py-1">
+      <div class="nav-left flex-gap-2">
+        <div class="brand-container flex-center-y flex-gap">
+          <img class="brand-logo" src="./media/logo-cap.png" alt="Education" />
+          <div class="h3">E Tutor</div>
+        </div>
+      </div>
+
+      <div class="nav-right flex-gap-2">
+        <a href="#"><i class="far fa-heart"></i></a>
+        <a href="#"><i class="far fa-bell"></i></a>
+        <a href="#"><i class="fas fa-cart-plus"></i></a>
+        <a href="#">
+          <img
+            class="avatar avatar-sm"
+            src="https://picsum.photos/200/300/"
+            alt="avatar"
+          />
+        </a>
+      </div>
+    </nav>
+
+    <h2 class="h2 text-center">Checkout Page</h2>
+    <div class="flex flex-center">
+      <section class="card">
+        <div class="text-lg">Courses 03</div>
+        <div class="card-body flex-column flex-gap">
+          <div class="horizantal-nested-card flex flex-gap-2">
+            <img loading="lazy" class="media h-100" src="./media/course1.jpg" />
+            <div class="description flex-column flex-gap-0">
+              <div class="course-by text-sm">Courtney Henry</div>
+              <div class="h4">
+                Graphic Design Masterclass - Learn Great Design
+              </div>
+              <span class="warning h4">&#8377; 1300</span>
+            </div>
+          </div>
+          <div class="horizantal-nested-card flex flex-gap-2">
+            <img loading="lazy" class="media h-100" src="./media/course2.jpg" />
+            <div class="description flex-column flex-gap-0">
+              <div class="course-by text-sm">Courtney Henry</div>
+              <div class="h4">
+                Graphic Design Masterclass - Learn Great Design
+              </div>
+              <span class="warning h4">&#8377; 1300</span>
+            </div>
+          </div>
+          <div class="horizantal-nested-card flex flex-gap-2">
+            <img loading="lazy" class="media h-100" src="./media/course3.jpg" />
+            <div class="description flex-column flex-gap-0">
+              <div class="course-by text-sm">Courtney Henry</div>
+              <div class="h4">
+                Graphic Design Masterclass - Learn Great Design
+              </div>
+              <span class="warning h4">&#8377; 1300</span>
+            </div>
+          </div>
+        </div>
+        <div class="card-footer">
+          <div class="h4">Order Summary</div>
```

Comment:
Here why not use <h4> tag instead of using <div>

Reply:
h4 comes with margin, so I avoid. In this case I will fix